website/docs/r/ipset.html.markdown

```diff
@@ -0,0 +1,64 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_ipset"
+sidebar_current: "docs-vcd-resource-ipset"
+description: |-
+  Provides an IP set resource.
+---
+
+# vcd\_ipset
+
+Provides a vCloud Director IP set resource. An IP set is a group of IP addresses that you can add as
+  the source or destination in a firewall rule or in DHCP relay configuration.
+
+
+Supported in provider *v2.6+*
+
+## Example Usage 1
```

Comment:
Is there an example 2 coming?
I'd like to see several examples of different combinations allowed in `ip_addresses` and at least one example of how to use this in other resources, even if they don't exist yet.

Reply:
it will come either in IP set or in vcd_nsxv_firewall_rule when there is a PR to start supporting it.